pages/dkp/kommander/1.4/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: layout.pug
+navigationTitle: Requirements for Attaching an Existing Cluster to Kommander
+title: Requirements for Attaching an Existing Cluster to Kommander
+menuWeight: 3
+excerpt: Requirements for attaching an existing cluster to Kommander
+beta: true
+---
+
+## Basic requirements
+
+To attach an existing cluster to Kommander, the Kommander management cluster must be able to reach the services and the api-server of the target cluster.
+
+You can attach existing clusters with or without networking restrictions to Kommander. These networking restrictions refer to clusters that are located in a DMZ, behind a proxy server or a firewall, or that have additional requirements for access.
+
+<p class="message--note"><strong>NOTE: </strong>Kommander does not support attachment of K3s clusters.</p>
+
+For [attaching existing clusters _without_ networking restrictions][attach_without_network_restrictions], the requirements depend on which Kommander 1.X version you are using. Each version of Kommander supports a specific range of Kubernetes versions. You must ensure that the target cluster is running a compatible version.
```

Comment:
I think we can just get rid of this `1.x` business here. It just matters on the version.

```suggestion
For [attaching existing clusters _without_ networking restrictions][attach_without_network_restrictions], the requirements depend on which Kommander version you are using. Each version of Kommander supports a specific range of Kubernetes versions. You must ensure that the target cluster is running a compatible version.
```

Reply:
Resolved as noted in my branch.